Python/ceval.c

```diff
@@ -2989,3 +2990,16 @@ void Py_LeaveRecursiveCall(void)
 {
     _Py_LeaveRecursiveCall();
 }
+
+PyObject *
+_PyCompile_GetCommonConstantsList(void)
```

Comment:
Is ceval.c the right place for this function? Maybe this could be in Python/intrinsics.c?

Reply:
Happy to put it in intrinsics.c if you prefer that. I put it here instead of intrinsics.c because the implementation of the opcode doesn't use intrinsics.c.

Reply:
Also, the function relies on a variable defined in `ceval_macros.h`, which isn't included by intrinsics.c. However, it looks like I'll have to create the array differently on Windows.

Reply:
I think this fits in intrinsics (intrinsic constants similar to intrinsic functions). Open to objections.

Reply:
This function no longer exists.